### PR TITLE
lua: add flag to selectively enable modules and symbols

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

File: script/module.go [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+		for _, name := range m.disabledSymbols {
+			st.RawSetString(name, lua.LNil)
+		}
+	}
+}
+
+// withSymbols returns copy of module with selected symbols
+func (m luaModule) withSymbols(L *lua.LState, enabledSymbols []string) luaModule {
+	// gopher-lua does not have API to select enabled symbols,
+	// see https://github.com/yuin/gopher-lua/discussions/393
```
Comment (AlexanderYastrebov) [inline script/module.go:54]: https://github.com/yuin/gopher-lua/discussions/393

**Comment by szuecs** [comment]
@AlexanderYastrebov should we have a default to only load a subset of modules and increase minor version to explain that this would break in case you use lua filter and you need to check the release/docs pages?
I think a subset would be something like http/json/base64 modules and maybe some string functions, which are safe to use.

**Comment by AlexanderYastrebov** [comment]
This change is backward compatible. If we decide to do a breaking change then I would rather disable lua altogether by default

**Comment by szuecs** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**